stock_eggs/source-engine/egg-counter--strike--global-offensive.json

```diff
@@ -1,14 +1,19 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v1",
+        "update_url": null
     },
-    "exported_at": "2020-10-19T23:29:57+00:00",
-    "name": "Counter-Strike: Global Offensive",
+    "exported_at": "2022-05-03T19:37:43+05:30",
+    "name": "csgo",
     "author": "support@pterodactyl.io",
     "description": "Counter-Strike: Global Offensive is a multiplayer first-person shooter video game developed by Hidden Path Entertainment and Valve Corporation.",
-    "image": "quay.io\/pterodactyl\/core:source",
-    "startup": ".\/srcds_run -game csgo -console -port {{SERVER_PORT}} +ip 0.0.0.0 +map {{SRCDS_MAP}} -strictportbind -norestart +sv_setsteamaccount {{STEAM_ACC}}",
+    "features": null,
+    "images": [
+        "quay.io\/pterodactyl\/core:source"
+    ],
+    "file_denylist": [],
+    "startup": ".\/srcds_run -game csgo -console -strictportbind -usercon +ip 0.0.0.0 -port {{SERVER_PORT}} +clientport 27005 +tv_port {{TV_PORT}} +sv_setsteamaccount {{STEAM_ACC}} -maxplayers_override {{MAXPLAYERS}} -tickrate {{TICRATE}} +map {{SRCDS_MAP}} +mapgroup {{MAPGROUP}} +game_type {{GAMETYPE}} +game_mode {{GAMEMODE}} +host_workshop_collection {{WS_COLLECTIONID}} +workshop_start_map {{WS_MAP}} -authkey {{WS_APIKEY}} -nobreakpad -autoupdate -steam_dir steamcmd -steamcmd_script update.txt",
```

Comment:
Forcing the client port is a no-go.

Reply:
If a variable is not set will it error out as the next value will be another flag.